pipeline_dp/budget_accounting.py

```diff
@@ -285,29 +291,32 @@ class NaiveBudgetAccountant(BudgetAccountant):
     def __init__(self,
                  total_epsilon: float,
                  total_delta: float,
-                 n_aggregations: Optional[int] = None,
+                 num_aggregations: Optional[int] = None,
                  aggregation_weights: Optional[list] = None):
         """Constructs a NaiveBudgetAccountant.
 
         Args:
             total_epsilon: epsilon for the entire pipeline.
             total_delta: delta for the entire pipeline.
-            n_aggregations: number of aggregations for which 'self' manages
-              the budget. All aggregations should have budget_weight = 1.
-              If None, any number of aggregations is allowed.
-            aggregation_weights: weights of aggregations for which 'self'
-              manages the budget. If None, any number of aggregations and
-              weights are allowed.
+            num_aggregations: number of DP aggregations in the pipeline for
+             which  'self' manages the budget. All aggregations should have
+             'budget_weight' = 1. When specified all aggregations will have
+             equal budget. It is useful to ensure that the pipeline has fixed
+             number of DP aggregations.
+            aggregation_weights: 'budget_weight' of aggregations for which
+             'self' manages the budget. It is useful to ensure that the pipeline
+              has a fixed number of DP aggregations with fixed weights.
+
+        if num_aggregations and aggregation_weights are not set, there are no
+        restrictions on number of aggregations nor their budget_weights
```

Comment:
nit: "on *the* number of aggregations"

nit: "budget weights" instead of "budget_weights"?